public/doc/openapi-3-v1.0.0.json

```diff
@@ -978,6 +901,15 @@
                     "type": "object"
                 }
             },
+            "resource_object": {
+                "in": "query",
+                "name": "resource_object",
```

Comment:
@hsong-rh Is this a nested query parameter?
Shouldn't it be 3 different parameters one for app_name, object_type, object_id?

Reply:
@mkanoor No, I just changed it into nested parameters, as Bill suggested.

Reply:
@hsong-rh @bzwei 
Have you seen any examples of creating nested parameters, in the URL format you can only pass non nested separate query parameters. I haven't seen ways to group parameters together. We have to ensure that this would work thru curl.

Reply:
I wanted @hsong-rh to follow the example of the filter parameter.

Reply:
@bzwei By keeping 3 separate query parameters we can have the Approval do the following
- Only app_name is specified List all workflows associated with the the app_name (micro service)
- When app_name and object_type is specified List all workflows related to the app_name and object type (all workflows for Topology and ServiceInventory)
- When app_name, object_type and object_id are specified list all workflows related to a specific object from a specific service. This requires us to go out to the remote service and collect the tags and then filter out the associated workflows.

Reply:
@mkanoor Do we have a use case that requires mentioned queries? Only the third one is implemented in the PR.
If the three attributes are requests' native columns what you suggested is legitimate, but they belong to tag_links. We'd better to have a dedicated key to indicate they are not build-in attributes like the filter parameter does. Otherwise how do we distinguish if in the future we need to add a column called `object_type` or `object_id`?

Reply:
@bzwei Either way we need to validate this change with a generated client code to ensure that it can serialize the nested query parameter. So lets generate a Approval Client and test the nested parameter and see how it escapes it.

Reply:
@mkanoor @bzwei 
I tested with both Curl and Postman. The query parameters in our case has to be defined in body for GET action. Based on our discussion, I will change back to use three separate query parameters, instead of current single `resource_object`